src/tests/integration/test-data/show/AddConst.applyOp(uint256).cse.expected

```diff
@@ -6,7 +6,7 @@
 │   statusCode: STATUSCODE:StatusCode
 ┃
 ┃ (branch)
-┣━━┓ constraint: VV0_x_114b9705:Int <=Int ( maxUInt256 -Int #lookup ( CONTRACT_STORAGE:Map , 0 ) )
+┣━━┓ constraint: VV0_x_114b9705:Int <=Int ( maxUInt256 -Int #lookup ( STORAGE_3b3c4496:Map , 0 ) )
```

Comment:
How is the suffix `3b3c4496` created, do we know? Will it be stable?

Reply:
It's created by `abstract_cell_safely()`. I'm not sure what you mean by stable.

Reply:
I meant that it looks like a hash of some kind, and if it changes in future the expected outputs will also change.

Reply:
That function is implemented as a hash of the term that's being abstracted, and it will rehash if there is still a name collision. I've replaced it with a system that generates one from either the variable name of the contract's ID if it's a variable or the actual ID.